test/integration/cannot_view_another_users_dashboard_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.